setup.py

```diff
@@ -480,7 +480,7 @@ def get_file_list (self):
     },
     # Requirements, usable with setuptools or the new Python packaging module.
     install_requires = [
-        'requests >= 2.2.0',
+        'requests<2.15,>=2.2',
```

Comment:
@anarcat  yeah it just "shortened it" :smile:  linkchecker + new requests doesn't work for some reason (but it can be specific for https://www.alpinelinux.org/)